instance/models/openedx_appserver.py

```diff
@@ -144,6 +154,8 @@ def set_field_defaults(self):
         """
         Set default values.
         """
+        if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:
```

Comment:
@smarnach I understood @bradenmacdonald 's comment as being better to keep it like @haikuginger wrote it above (`if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION`) - that wouldn't create a migration?

In any case, I think that if any users have access but no list of organization is set, we should set the default team - I agree with @bradenmacdonald that we will almost always want `DEFAULT_ADMIN_ORGANIZATION` to be included on the list .